torch/_dynamo/variables/builder.py

```diff
@@ -811,7 +799,9 @@ def wrap_unspecialized_primitive(self, value):
                 self.tx.output.tracked_fakes.append(
                     TrackedFake(wrapped_value, self.source)
                 )
-                # TODO: Do float
+                # TODO: Do float?
```

Comment:
I think this comment can be removed with the current plan?

Reply:
yes